src/test/objectstore/store_test.cc

```diff
@@ -494,16 +496,18 @@ TEST_P(StoreTest, FiemapHoles) {
     // kstore always return [0, object_size] regardless of offset and length
     // FIXME: if fiemap logic in kstore is refined
     if (string(GetParam()) != "kstore") {
+      last = --m.end();
       ASSERT_GE(m[SKIP_STEP], 3u);
+    
+      extents_exist = true;
+      if (m.size() == (MAX_EXTENTS - 2)) {
+        for (uint64_t i = 1; i < MAX_EXTENTS - 1; i++)
+	  extents_exist = extents_exist && m.count(SKIP_STEP*i);
+      }
+      ASSERT_TRUE((m.size() == 1 &&
+		   m[SKIP_STEP] > SKIP_STEP * (MAX_EXTENTS - 2)) ||
```

Comment:
i think the offset is not greater than the where the extend begins, `offset + len` is.